control-plane/outputs.tf

```diff
@@ -53,3 +53,52 @@ output "k3s_cluster_secret" {
   sensitive = true
 }
 
+output "k3s_tls_san" {
+  value     = module.k3s.k3s_tls_san
+}
+
+output "db_engine_version" {
+  value     = var.db_engine_version
+}
+
+output "db_skip_final_snapshot" {
+  value     = var.db_skip_final_snapshot
+}
+
+output "server_k3s_exec" {
+  value     = var.server_k3s_exec
+}
+
+output "install_certmanager" {
+  value     = var.install_certmanager
+}
+
+output "certmanager_version" {
+  value     = var.certmanager_version
+}
+
+output "tls_cert_file" {
+  value     = module.k3s.tls_cert_file
+  sensitive = true
+}
+
+output "tls_key_file" {
+  value     = module.k3s.tls_key_file
+  sensitive = true
+}
+
+output "random_prefix" {
+  value     = var.random_prefix
+}
+
+output "rancher_chart_tag" {
+  value     = var.rancher_chart_tag
+}
+
+output "rancher_version" {
+  value     = var.rancher_version
+}
+
+output "use_new_bootstrap" {
+  value = module.k3s.use_new_bootstrap
+}
```

Comment:
Is the formatting correct on all of these? Looks awkward that the single line ones have additional whitespace.

Reply:
Fair, I'll fix it. It was likely done by the terraform extension in vscode.